R/tax_range_time.R

```diff
@@ -157,7 +173,12 @@ tax_range_time <- function(occdf,
     if (by == "LAD") {
       temp_df <- temp_df[order(temp_df$min_ma), ]
       temp_df$taxon_id <- seq_len(nrow(temp_df))
-      }
+    }
+
+    if (is.null(group)) {
+      temp_df <- temp_df[order(temp_df$group), ]
+      temp_df$taxon_id <- seq_len(nrow(temp_df))
+    }
 
     # Plot data?
     if (plot == TRUE) {
```

Comment:
When `group` is supplied, is it worth plotting different groups by different colours?